client/src/interface/User.ts

```diff
@@ -1,6 +1,5 @@
 export interface User {
   email: string;
-  username: string;
```

Comment:
Why are we removing username here? I think we need it.

Reply:
The mock-up does not have a username field.

Reply:
Ah ok, I think we'll have to make this change on the backend as well then.

Reply:
Yes this is part of Felix's task. Felix please make sure all of the auth functionalities are still functional in this PR